astropy/time/tests/test_update_leap_seconds.py

```diff
@@ -81,3 +83,17 @@ def test_auto_update_expired_file(self, tmpdir):
 
         with pytest.warns(iers.IERSStaleWarning):
             update_leap_seconds(['erfa', expired_file])
+
+    def test_init_thread_safety(self, monkeypatch):
+        # Set up expired ERFA leap seconds.
+        expired = self.erfa_ls[self.erfa_ls['year'] < 2017]
+        expired.update_erfa_leap_seconds(initialize_erfa='empty')
```

Comment:
For any possible other reviewer, the leap second table proper is safeguarded in the test class `setup`/`teardown` methods.